### PR TITLE
i18n(fr): Update `guides/markdown-content.mdx` from #9888

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
[i18n(fr): Update](https://github.com/withastro/docs/commit/45c79871ea60540ce9947f2829c18ffc1ea1ef72) [guides/markdown-content.mdx](https://github.com/withastro/docs/commit/45c79871ea60540ce9947f2829c18ffc1ea1ef72) [from](https://github.com/withastro/docs/commit/45c79871ea60540ce9947f2829c18ffc1ea1ef72) https://github.com/withastro/docs/pull/9888

> [!NOTE]
> Merge #9934 before this one. Links are broken.

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 1]
```diff
@@ -502,3 +378,21 @@ const {frontmatter} = Astro.props;
 Vous pouvez également [styliser votre Markdown](/fr/guides/styling/#style-markdown) dans votre composant de mise en page.
 
 <ReadMore>En savoir plus sur les [Mises en page Markdown](/fr/basics/layouts/#mises-en-page-markdown).</ReadMore>
+
+## Récupérer du Markdown à distance
+**Astro n'inclut pas de support intégré pour le Markdown à distance en dehors des [collections de contenu expérimentales](/fr/reference/configuration-reference/#experimentalcontentlayer) !**
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/markdown-content.mdx:383]: Missing blank lines:
```suggestion

**Astro n'inclut pas de support intégré pour le Markdown à distance en dehors des [collections de contenu expérimentales](/fr/reference/configuration-reference/#experimentalcontentlayer) !**

```

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 2]
```diff
@@ -502,3 +378,21 @@ const {frontmatter} = Astro.props;
...
+Pour récupérer directement du Markdown distant et le rendre au format HTML, vous devrez installer et configurer votre propre interpréteur de Markdown à partir de NPM. Celui-ci **n'héritera pas** des paramètres Markdown intégrés d'Astro que vous avez configurés.
+Assurez-vous de bien comprendre ces limitations avant d'implémenter ceci dans votre projet, et envisagez de récupérer votre Markdown distant en utilisant un chargeur de collections de contenu à la place.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/markdown-content.mdx:385]: Missing blank line:
```suggestion

Assurez-vous de bien comprendre ces limitations avant d'implémenter ceci dans votre projet, et envisagez de récupérer votre Markdown distant en utilisant un chargeur de collections de contenu à la place.
```

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 3]
```diff
@@ -502,3 +378,21 @@ const {frontmatter} = Astro.props;
...
+
+```astro title="src/pages/remote-example.astro"
+---
+// Exemple: Récupération du contenu Markdown depuis une API distante
+// et le rendre au format HTML, au moment de l'exécution.
+// Using "marked" (https://github.com/markedjs/marked)
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/markdown-content.mdx:391]: Missing space before colon and the last comment line was not translated:
```suggestion
// Exemple : Récupération du contenu Markdown depuis une API distante
// et le rendre au format HTML, au moment de l'exécution.
// En utilisant "marked" (https://github.com/markedjs/marked)
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Some blank lines are missing which makes it a single block of text rather than multiple paragraphs.

**Comment by thomasbnt** [comment]
> Some blank lines are missing which makes it a single block of text rather than multiple paragraphs.

Owh again.. thanks @ArmandPhilippot,I've had trouble since updating the comparison on GitHub. I'll have to see how I can see line breaks better. 
JakePeraltaSquintingGIF

**Comment by ArmandPhilippot** [comment]
@thomasbnt No problem! 😄 

I was checking the broken links, it seems that `upgrade-to/v1.mdx` is not marked as outdated because it was partially updated... But we need an update around l333 to fix the link issue, see:
* VO: https://github.com/withastro/docs/pull/9888/files#diff-603be2c6308fb91013f4800ce1da883269c0a0b6e341c0bacbf5d5b24eaddedf
* VF: https://github.com/withastro/docs/pull/9888/files#diff-7614c40a5e95e4c2db15c8dfba732461efd38189d676619fdce9965b640a4da1

Can you add those changes to this PR?

As you noted the others are already corrected, so after that we'll be good! 🎉

**Review by ArmandPhilippot — APPROVED** [review]
Ah well seen about the other line, I thought it was already up to date!

LGTM! 🚀

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**